src/Silex/Provider/StatsdServiceProvider.php

```diff
@@ -18,10 +19,9 @@ class StatsdServiceProvider implements ServiceProviderInterface
      * Register Service Provider
      * @param Application $app Silex application instance
```

Comment:
The variable type has now changed, could you please also update the documentation here